src/vm/amd64/umthunkstub.S

```diff
@@ -0,0 +1,567 @@
+//
+// Copyright (c) Microsoft. All rights reserved.
+// Copyright (c) Geoff Norton. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information. 
+//
+
+// ==++==
+//
+
+//
+// ==--==
+
+#include "unixasmmacros.inc"
+#include "asmconstants.h"
+
+#ifdef FEATURE_MIXEDMODE
```

Comment:
FEATURE_MIXEDMODE is not supported on Linux/Mac, so we can safely remove all code from this file that is guarded by this define.